### PR TITLE
Quiz rules page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added Quiz Rules Static page.
Screenshot (325)

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
good to merge. remove unnecessary commented code

File: src/Pages/Rules/QuizRules.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          <div className="quiz-rules__button--container">
+            <button
+              className=" quiz-rules__btn quiz-rules__button--primary"
+              onClick={() => navigate("/")}
+            >
+              Back
+            </button>
+            <button
+              className="quiz-rules__btn quiz-rules__button--secondary"
+              onClick={() => navigate("/questions/")}
```
Comment (munjalSaksham71) [inline src/Pages/Rules/QuizRules.jsx]: replace ```/questions/```  to ```/question```

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -22,7 +24,12 @@ export function Card() {
                   </div>
                 </div>
                 <div className="card__secondary--actions">
-                  <button className="card__btn card__button--primary">
+                  <button
+                    className="card__btn card__button--primary"
+                    onClick={() => {
+                      navigate("/rules/");
```
Comment (munjalSaksham71) [inline src/components/Card/Card.jsx]: Replace as mentioned above

File: src/App.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import "./App.css";
 import { Home } from "../src/Pages/Home-Page/Home";
 // import { Login } from "../src/Pages/Login-Page/Login";
 // import { Signup } from "../src/Pages/Signup-Page/Signup";
-// import { QuizRules } from "./Pages/Rules/QuizRules";
+import { QuizRules } from "./Pages/Rules/QuizRules";
 // import { QuizQuestions } from "./Pages/Question-Page/QuizQuestions";
 // import { Report } from "./Pages/Report-Page/Report";
 // import { Result } from "./Pages/Result-Page/Result";
```
Comment (munjalSaksham71) [inline src/App.js:8]: Remove these unnecessary comments

**Review by munjalSaksham71 — APPROVED** [review]
Just added few suggestion and then you are good to merge

**Comment by SaurabhChawda** [comment]
> good to merge. remove unnecessary commented code

Ok, ThankYou

File: src/Pages/Rules/QuizRules.jsx (hunk 1 above)
Comment (SaurabhChawda) [reply src/Pages/Rules/QuizRules.jsx]: updated, Thankyou

**Pull request merged**